src/interface.rs

```diff
@@ -162,6 +162,13 @@ pub fn expr_to_name(expr: &swc_ecmascript::ast::Expr) -> String {
       let right = expr_to_name(&*member_expr.prop);
       format!("[{}.{}]", left, right)
     }
+    Lit(lit) => {
+      use swc_ecmascript::ast::Lit::*;
+      match lit {
+        Str(str) => str.value.to_string(),
+        _ => "<TODO>".to_string(),
+      }
+    }
```

Comment:
You could use code from `deno_lint` to handle all literals:
https://github.com/denoland/deno_lint/blob/1a1718b7eac09d60e57cf291418471e551a3f6c5/src/swc_util.rs#L68-L88